packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -357,18 +454,9 @@ export default class ClickableBehavior extends React.Component<
             (triggerOnEnter && keyCode === keyCodes.enter) ||
             (triggerOnSpace && keyCode === keyCodes.space)
         ) {
-            // NOTE: In Firefox, the keyup event needs to be preventDefault-ed
-            // for space.
-            // https://stackoverflow.com/questions/24383722/unable-to-prevent-spacebar-default-action-in-firefox
-            if (keyCode === keyCodes.space) {
-                e.preventDefault();
-            }
```

Comment:
I don't see any harm in always calling `e.preventDefault()` since we call it inside of `runCallbackAndMaybeNavigate` and if we always called it here then we may as well just move the call inside of `runCallbackAndMaybeNavigate` and be done with it.